docs/installing-solidity.rst

```diff
@@ -197,7 +197,13 @@ Building Solidity is quite similar on Linux, macOS and other Unices:
     cd build
     cmake .. && make
 
-And even on Windows:
+or even easier:
+
+.. code:: bash
+
+    ./scripts/build.sh --Release #specifies the build type
```

Comment:
I think it should be just `Release` but in general, I guess it is better not to confuse the readers here and just mention the script instead.